browsertests/testutils_test.go

```diff
@@ -0,0 +1,216 @@
+package browser_test
+
```

Comment:
The name of this file testutils_test.go is a bit weird since this file is not a test. I think I'd name it either testutils.go or browsertestsutil.go or something like that.